src/utils/openmc/material.py

```diff
@@ -76,7 +76,9 @@ def set_id(self, material_id=None):
 
         # If the Material already has an ID, remove it from global list
         if not self._id is None:
-            MATERIAL_IDS.remove(self._id)
+            try:
```

Comment:
Perhaps this ``try-catch`` block should be replicated for other class' ``set_id(...)`` routines for uniformity? Perhaps ``Universe``, ``Cell``, ``Lattice``, etc.